src/Options/Auto.hpp

```diff
@@ -10,12 +10,16 @@
 
 #include "Options/Options.hpp"
 #include "Utilities/GetOutput.hpp"
+#include "Utilities/PrettyType.hpp"
 
 namespace Options {
 /// The label representing the absence of a value for `Options::Auto`
-enum class AutoLabel { Auto, None };
-
-std::ostream& operator<<(std::ostream& os, AutoLabel label) noexcept;
+namespace AutoLabel {
+/// Automatically determine the default label
```

Comment:
This doesn't automatically determine the label, it just reads 'Auto' in the input file :) So just write `'Auto' label`